merge-upstream.yml 这两天失败了

```
Run git pull https://github.com/coolsnowwolf/lede.git --log --no-commit
From https://github.com/coolsnowwolf/lede
 * branch                HEAD       -> FETCH_HEAD
hint: You have divergent branches and need to specify how to reconcile them.
hint: You can do so by running one of the following commands sometime before
hint: your next pull:
hint: 
hint:   git config pull.rebase false  # merge (the default strategy)
hint:   git config pull.rebase true   # rebase
hint:   git config pull.ff only       # fast-forward only
hint: 
hint: You can replace "git config" with "git config --global" to set a default
hint: preference for all repositories. You can also pass --rebase, --no-rebase,
hint: or --ff-only on the command line to override the configured default per
hint: invocation.
fatal: Need to specify how to reconcile divergent branches.
Error: Process completed with exit code 128.
````

这两天更新都是失败，等大佬修复

2021-10-21T20:54:33.1068950Z Reset branch 'master'
2021-10-21T20:54:33.1069836Z Branch 'master' set up to track remote branch 'master' from 'origin'.
2021-10-21T20:54:33.2154054Z ##[endgroup]
2021-10-21T20:54:33.2202140Z [command]/usr/bin/git log -1 --format='%H'
2021-10-21T20:54:33.2229198Z '679dcacac5a6f6b8a8fb5719473fd4b6c2605d30'
2021-10-21T20:54:33.2363655Z ##[group]Run git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
2021-10-21T20:54:33.2364995Z [36;1mgit config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"[0m
2021-10-21T20:54:33.2366101Z [36;1mgit config --global user.name "github-actions[bot]"[0m
2021-10-21T20:54:33.2423020Z shell: /usr/bin/bash -e {0}
2021-10-21T20:54:33.2423457Z ##[endgroup]
2021-10-21T20:54:33.2582376Z ##[group]Run git pull https://github.com/coolsnowwolf/lede.git --log --no-commit
2021-10-21T20:54:33.2583343Z [36;1mgit pull https://github.com/coolsnowwolf/lede.git --log --no-commit[0m
2021-10-21T20:54:33.2624356Z shell: /usr/bin/bash -e {0}
2021-10-21T20:54:33.2624778Z ##[endgroup]
2021-10-21T20:54:33.6713222Z From https://github.com/coolsnowwolf/lede
2021-10-21T20:54:33.6714550Z  * branch                HEAD       -> FETCH_HEAD
2021-10-21T20:54:33.6804356Z hint: You have divergent branches and need to specify how to reconcile them.
2021-10-21T20:54:33.6805534Z hint: You can do so by running one of the following commands sometime before
2021-10-21T20:54:33.6806378Z hint: your next pull:
2021-10-21T20:54:33.6807051Z hint: 
2021-10-21T20:54:33.6807743Z hint:   git config pull.rebase false  # merge (the default strategy)
2021-10-21T20:54:33.6808408Z hint:   git config pull.rebase true   # rebase
2021-10-21T20:54:33.6809371Z hint:   git config pull.ff only       # fast-forward only
2021-10-21T20:54:33.6809824Z hint: 
2021-10-21T20:54:33.6810761Z hint: You can replace "git config" with "git config --global" to set a default
2021-10-21T20:54:33.6811931Z hint: preference for all repositories. You can also pass --rebase, --no-rebase,
2021-10-21T20:54:33.6813012Z hint: or --ff-only on the command line to override the configured default per
2021-10-21T20:54:33.6813665Z hint: invocation.
2021-10-21T20:54:33.6814260Z fatal: Need to specify how to reconcile divergent branches.
2021-10-21T20:54:33.6828344Z ##[error]Process completed with exit code 128.
2021-10-21T20:54:33.6947959Z Post job cleanup.
2021-10-21T20:54:33.8093765Z [command]/usr/bin/git version
2021-10-21T20:54:33.8142339Z git version 2.33.1
2021-10-21T20:54:33.8187138Z [command]/usr/bin/git config --local --name-only --get-regexp core\.sshCommand
2021-10-21T20:54:33.8222540Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :
2021-10-21T20:54:33.8520801Z [command]/usr/bin/git config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2021-10-21T20:54:33.8546950Z http.https://github.com/.extraheader
2021-10-21T20:54:33.8557788Z [command]/usr/bin/git config --local --unset-all http.https://github.com/.extraheader
2021-10-21T20:54:33.8599568Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :
2021-10-21T20:54:33.8957754Z Cleaning up orphan processes

--- a/merge-upstream.yml
+++ b/merge-upstream.yml
@@ -1,7 +1,7 @@
 #
 # This is free software, lisence use MIT.
 # 
-# Copyright (C) 2020 KFERMercer <KFER.Mercer@gmail.com>
+# Copyright (C) 2021 KFERMercer <KFER.Mercer@gmail.com>
 # 
 # <https://github.com/KFERMercer/OpenWrt-CI>
 #
@@ -10,7 +10,7 @@ name: Merge-upstream
 
 on:
   push:
-    branches: 
+    branches:
       - master
   schedule:
     - cron: 30 19 * * *
@@ -30,10 +30,12 @@ jobs:
         fetch-depth: 0
         lfs: true
 
-    - name: Set git identity
+    - name: Set git config
       run : |
         git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
         git config --global user.name "github-actions[bot]"
+        git config --global pull.rebase false
+        git config --global --list
 
     - name: Load upstream commits
       run: git pull https://github.com/coolsnowwolf/lede.git --log --no-commit
@@ -43,7 +45,7 @@ jobs:
         if [ -f ./.git/MERGE_MSG ]; then
         mkdir ./tmp && cp ./.git/MERGE_MSG ./tmp/message
         sed -i "1c [bot] AutoMerging: merge all upstream's changes:" ./tmp/message
-        sed -i '/^\#.*/d' ./tmp/message
+        # sed -i '/^\#.*/d' ./tmp/message
         git commit --file="./tmp/message"
         else
         echo "There is no merge commits."

